tests/test_integration.py

```
@@ -123,7 +123,48 @@ async def run_integration_test():
         )
         print(f"Set work item 1 as parent of work item 2")
 
-        # 5. Delete work items
+        # 5. Create epic with work item 1 as the underlying work item
+        print(f"Creating epic...")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)